### DISCUSSION TITLE
row.cell with Image and caption

### DISCUSSION THREAD

**Discussion by R00118189** [py-pdf/fpdf2#1274] [Q&A]
I am trying to add caption to a cell containing image
`row.cell(img=self.get_photo(photo),img_fill_width=True)` but want to add additional info directly below the image, ex. data from picture exif metadata like `DateTimeOriginal` is this possible with fpdf2?

**Comment by Lucas-C** [comment]
Hi @R00118189

For now, no, this is not possible using `fpdf2` and tables: https://py-pdf.github.io/fpdf2/Tables.html.

However there is an in-progress work by @gmischler to support this use-case: https://github.com/py-pdf/fpdf2/pull/1200

You can still use custom code, either subclassing the [Table class](https://py-pdf.github.io/fpdf2/fpdf/table.html#fpdf.table.Table) or just using [FPDF.multi_cell()](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.multi_cell)
